lib/challenge.js

```diff
@@ -20,6 +20,14 @@
 
 // Answer Starts Here
 
+var alarm clock = {
+    display: 'digital',
+    color: 'black',
+    weight: '8 ounces',
+    togglePowerSource: ['AC', 'battery']
+    togglePower: ['on', 'off']
+    setAlarm: ['time', 'noise']
+}
```

Comment:
This is not at all what the question was asking for, but your enthusiasm for writing object literals should be rewarded. 1/2